src/components/Picker/Picker.js

```diff
@@ -151,6 +157,18 @@ class Picker extends PureComponent {
         this.props.onInputChange(this.props.items[0].value, 0);
     }
 
+    setIsOpen() {
+        this.setState({
+            isOpen: true,
+        });
+    }
+
+    setIsClosed() {
+        this.setState({
+            isOpen: false,
+        });
+    }
```

Comment:
You are right, I think we should clarify what that state variable actually controls.  Since it is only responsible for enabling and disabling the highlight, we can make it more understandable by using an `isHighlighted` state variable, and calling `enableHighlight` and `disableHighlight` to control it.